src/main/java/com/algorithmia/algorithmHandler/ResponseHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.algorithmia.algorithmHandler;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+
+final class ResponseHandler {
+
+    private String FIFOPATH = "/tmp/algoout";
+
+    private PrintStream output;
+
+    ResponseHandler()throws java.io.IOException{
```

Comment:
Please catch and wrap the `IOException` throw in this method with a `RuntimeException` rather than throwing the bare `IOException` itself.